fem/pgridfunc.hpp

```diff
@@ -434,13 +434,23 @@ class ParGridFunction : public GridFunction
    /// be used for ASCII output.
    void Save(const char *fname, int precision=16) const override;
 
-   /// Returns a GridFunction on MPI rank @a save_rank that does not have any
-   /// duplication of vertices/nodes at processor boundaries.
-   /// serial_mesh is obtained using ParMesh::GetSerialMesh(save_rank).
-   /// Note that the @ save_rank argument must match for the
-   /// ParMesh::GetSerialMesh and GetSerialGridFunction method.
+   /// @brief Returns a GridFunction on MPI rank @a save_rank that does not have
+   /// any duplication of vertices/nodes at processor boundaries.
+   ///
+   /// The @a serial_mesh is obtained using ParMesh::GetSerialMesh(save_rank).
+   /// Note that the @a save_rank must be the same as that used in
+   /// ParMesh::GetSerialMesh.
```

Comment:
Also mention that the gridfunction owns the finite element space and collection?

Reply:
Done in afbd1a1ba590ade194e42b86f7873daaa23e6547.